script/package-worker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+NUGET_DIR=$1
+WORKER_DIR=$2
+OUTPUT_DIR=$3
+
+VERSION=`ls ${NUGET_DIR}/Microsoft.Spark.*.nupkg | head -n 1 | xargs -n 1 basename | cut -d'.' -f3-5`
```

Comment:
You might want to take a look at how I did this in [install-worker.sh](https://github.com/dotnet/spark/blob/master/deployment/install-worker.sh#L46) and potentially put in a version check in case the version string is not as expected? (just to be on the safe side)

Reply:
changed to using powershell.   Don't think we need to check this unless we are going to change the way the nuget filename is going to be named.